src/dask_awkward/core.py

```diff
@@ -591,8 +610,16 @@ def calculate_known_divisions(array: DaskAwkwardArray) -> tuple[int, ...]:
         Locations (indices) of division boundaries.
 
     """
+    # if divisions are known, quick return
     if array.known_divisions:
         return array.divisions  # type: ignore
+    # handle the case where we have 1 partition
+    if array.npartitions == 1:
```

Comment:
Do we need this special case? Note that this branch returns tuples rather than a list.

Reply:
Without the special case `np.cumsum` is failing because the `array.map_partitions(ak.num, axis=0)` returns an int instead of an array. I guess we can force the computation to be an array if it's not already?

Reply:
Always converting results to arrays now